### ISSUE TITLE
Add support for scanning local repository

### ISSUE THREAD

**Issue by erjoalgo** [dakrone/emacs-java-imports#15]
It would be nice to be able to use jars in the local repository to automatically provide defaults for class packages

**Comment by dakrone** [comment]
Closed by #16

**Comment by erjoalgo** [comment]
Thanks. It looks like `replace-regexp-in-string` still creates a dependency on the subr-x, while the travis test is using emacs pre-24.4. At the same I added back the s.el dependency. But even if I switched to `s-replace`, it is merely a proxy for `replace-regexp-in-string`:

```lisp
(defun s-replace (old new s)
  "Replaces OLD with NEW in S."
  (replace-regexp-in-string (regexp-quote old) new s t t))```

I can't remember the pre-emacs-24.4 way of doing this, and I'm not sure if it's worth upgrading the emacs version in the test, but I'm sure users of this function would be able to figure it out.
